lib/generators/rodauth/install_generator.rb

```diff
@@ -71,9 +74,14 @@ def create_mailer
         end
 
         def create_fixtures
-          test_unit_options = ::Rails.application.config.generators.options[:test_unit]
-          if test_unit_options[:fixture] && test_unit_options[:fixture_replacement].nil?
-            template "app/test/fixtures/accounts.yml"
+          test_unit_options = ::Rails.application.config.generators.options[:test_unit]          
+          if test_unit_options[:fixture] && test_unit_options[:fixture_replacement].nil?                                    
+            if defined?(RSpec::Rails)              
```

Comment:
Did you try checking the corrected generator option instead? Seems like it would be more robust, and testable.